### DISCUSSION TITLE
QUESTION: docker-mailserver SMTP_ONLY postfix-accounts.cf fixed?

### DISCUSSION THREAD

**Discussion by gdgellatly** [Tecnativa/doodba#353] [Q&A]
@Yajo I see a commit for the amavis issues.

In our internal docs we also have a note
https://github.com/tomav/docker-mailserver/issues/1359 
WHenever restarting mail need to run `docker-compose exec smtp touch /tmp/docker-mailserver/postfix-accounts.cf`

Is this still necessary?

**Comment by yajo** [comment]
I have automated that and haven't tested what happens without the workaround for a long time. The linked issue was closed due to age, not to a fix, so I guess this is still the case, yes.

But maybe you can try and report back?

**Comment by gdgellatly** [comment]
@Yajo For 2 weeks I have been thinking, Why did you delete my issue, maybe it isn't an issue anymore. Now I found it. Anyway, I actually need to do a few new builds in next couple of weeks so will check then.

**Reply by yajo** [reply]
Sorry, I didn't delete it. It's just that Github added the discussions feature and I reorganized stuff, moving question-issues to discussions. But it's still here 😊

**Comment by bosd** [comment]
This topic is a lifesaver! 
Have serached quite a bit, couldnt find a solution..
(somehow didn't show up in my search results docker-mailserver/docker-mailserver#1359
I have found this, but there is no solution and closed deu to age:
https://github.com/docker-mailserver/docker-mailserver/issues/1687

Appearantly it is not fixed.  (Experiencing this on a fresh build V14 instance)
This error:
```Can’t read /tmp/docker-mailserver/postfix-accounts.cf: No such file or directory```
 is still in the logs. But it does'nt prevent from sending mails. :+1: 

But the mailserver is is causing the CPU usage to go to 100% :crying_cat_face: 
It's flooding the debugger with lines like:

```

smtp_1  | 2021-04-22 17:41:55,269 INFO spawned: 'changedetector' with pid 3298
smtp_1  | 2021-04-22 17:41:55,281 INFO success: changedetector entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
smtp_1  | 2021-04-22 17:41:55,282 INFO exited: changedetector (exit status 0; expected)
smtp_1  | 2021-04-22 17:41:55,284 INFO spawned: 'changedetector' with pid 3300
smtp_1  | 2021-04-22 17:41:55,297 INFO success: changedetector entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
smtp_1  | 2021-04-22 17:41:55,298 INFO exited: changedetector (exit status 0; expected)
smtp_1  | 2021-04-22 17:41:55,299 INFO spawned: 'changedetector' with pid 3302
smtp_1  | 2021-04-22 17:41:55,312 INFO success: changedetector entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
smtp_1  | 2021-04-22 17:41:55,313 INFO exited: changedetector (exit status 0; expected)

```



Edit: Your tip to run  ```docker-compose exec smtp touch /tmp/docker-mailserver/postfix-accounts.cf``` fixes this :tada:  :champagne: 
W/o smtp 5% CPU
smtp flooding: 100% CPU
proper workting smtp 6% CPU
